src/tokenizer/__init__.py

```diff
@@ -59,10 +59,10 @@
     TokenStream,
 )
 from .abbrev import Abbreviations, ConfigError
-from .version import __version__
 
 __author__ = "Miðeind ehf"
 __copyright__ = "(C) 2022 Miðeind ehf."
+__version__ = "3.4.3"  # Remember to also update in pyproject.toml
```

Comment:
Ég sé skv. þessu: https://stackoverflow.com/questions/67085041/how-to-specify-version-in-only-one-place-when-using-pyproject-toml að það er hægt að nota importlib til þess að nálgast útgáfunúmer eftir að pakki hefur verið byggður. Annars er ég hræddur um að þetta muni gleymast.

Reply:
Nice and clean! Done.